roles/ucp/templates/ucp.j2

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+usage="$0 start|stop"
+if [ $# -ne 1 ]; then
+    echo USAGE: $usage
+    exit 1
+fi
+
+case $1 in
+start)
+    set -e
+
+    echo starting ucp as {{ run_as }} on {{ node_name }}[{{ node_addr }}]
+
+    if [[ "{{ run_as }}" == "master" ]]; then
+        out=$(/usr/bin/docker run --rm -t --name ucp \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            docker/ucp install --host-address={{ node_addr }} \
+              --image-version={{ ucp_version }})
+        echo ${out}
+
+        # copy out the instance ID
+        instanceId=$(echo ${out} | egrep -o 'UCP instance ID: [a-zA-Z0-9:_]*' | \
+            awk --field-separator='UCP instance ID: ' '{print $2}')
+        echo instance-id: $instanceId
+        echo ${instanceId} > {{ ucp_remote_dir }}/{{ ucp_instance_id_file }}
```

Comment:
you need t o double-quote uses of variables like this in your shell scripts. it's a good practice to avoid spaces breaking your code.